lib/resources/r4/procedure.yaml

```diff
@@ -49,3 +52,204 @@ fields:
       - display: Procedure Personnel Relation Type
         system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/388
         info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-388-procedure-personnel-relation-type
+
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "Procedure"
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure
+  note: resourceType must be Procedure.
```

Comment:
also can we change all of the notes to be lists?

Reply:
Updated in this [commit](https://github.com/cerner/fhir.cerner.com/pull/372/commits/edd713404a0347648bfd408e254be8a2d11466ce#diff-80d4a0c17acadba0cd26a39c2cfeaf8dR24).